src/tailwind/plugins/theme/gc-screens.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * 2 extra breakpoints + print support
+ * 3xl: 2xl + 2 x the screen edge margins
+ * 4xl: 2000px
+ */
+
+module.exports = function () {
+  return {
+    '3xl': '1700px',
+    '4xl': '2000px',
```

Comment:
I like big screens

Reply:
Me too 😄 We have some elements that are touching the edge only up to a certain reasonable point, thus these.